yt/utilities/on_demand_imports.py

```diff
@@ -455,13 +455,13 @@ def concat(self):
 class Firefly_imports(OnDemand):
```

Comment:
better to keep the name of the class in sync too
```suggestion
class firefly_imports(OnDemand):
```

Reply:
(note that you'll need to update line 469 as well)